### PR TITLE
Restructure doc on 'kinds of hubs'

### PR DESCRIPTION
**Pull request by yuvipanda**
- Focus on problem we will solve for an audience, rather than
  the kind of service we will offer.
- Elide some technical details, and trim the prose a little to
  make this snappier. Technical details can live in peace elsewhere.
- Based on https://hackmd.io/YYEn6c_YQjCNi165F2HT9Q.

The research hub documentation isn't properly set here, and needs to
be.

### CODE REVIEW

**Comment by choldgraf** [comment]
Ah very nice - I just pushed a few edits to your branch, let me know what you think!

File: content/infrastructure/kinds-of-hubs.md [hunk 1]
```diff
@@ -4,52 +4,60 @@ weight = 30
...
-2i2c Hubs are extremely flexible and customizable, and can generally meet most
+JupyterHubs are flexible and customizable, and can generally meet most
 use-cases in research and education. There are 4 kinds of JupyterHub configurations
-we have found to be very common, and might help you think about what kinda JupyterHub
-configuration you want.
+we have found to be very common. We put extra emphasis on supporting these
+use-cases, which we'll describe below.
 
-## JupyterHub for your class
+## Teaching and learning with data science
```
Comment (yuvipanda) [inline content/infrastructure/kinds-of-hubs.md]: I was thinking of framing all these as 'JupyterHub for X', where X is a user persona of sorts. So folks should be able to tell if there's something that fits them from a scan. For example, both someone running a small class and someone with 20000 undergrads can fit 'teaching and learning with data science'

**Comment by yuvipanda** [comment]
@choldgraf left one comment about subtitles, I think this is good to go otherwise!

**Comment by yuvipanda** [comment]
This LGTM, @choldgraf!

**Pull request merged**